pkg/common/prometheus/metrics.go

```diff
@@ -150,7 +150,7 @@ var (
 		// Creating more buckets for operations that takes few seconds and less buckets
 		// for those that are taking a long time. A Full Sync operation taking a long time is
 		// unexpected and we don't have to be accurate(just approximation is fine).
-		Buckets: []float64{1, 2, 3, 4, 5, 7, 10, 12, 15, 18, 20, 25, 30, 60, 120, 180, 300},
+		Buckets: []float64{2, 5, 10, 15, 20, 25, 30, 60, 120, 180},
```

Comment:
I don't know why we need a histogram metric to capture FullSync related metrics. I don't think we need to answer percentile queries for full sync. So, I'm fine with just the Gauge metric here. This comment is not related to the intent of this PR(reduce buckets). So, we can track this as a separate bug.

Reply:
I'm ok to keep this as a histogram metric as this metric has a single label - status, which can be pass/fail(less cardinality). I'm also ok to convert this as a Gauge metric since the histogram is an overkill for this problem.